src/NUnitFramework/tests/Internal/PlatformDetectionTests.cs

```diff
@@ -72,7 +72,7 @@ public class PlatformDetectionTests
                 bool shouldPass = false;
 
                 foreach( string platform in expected )
-                    if ( shouldPass = platform.ToLower() == testPlatform.ToLower() )
+                    if (shouldPass = platform.ToUpperInvariant() == testPlatform.ToUpperInvariant())
```

Comment:
I think `testPlatform.Equals(platform, StringComparison.InvariantCultureIgnoreCase)` is superior because it states the actual goal and also doesn't allocate and transform temporary strings. Is there a reason we can't do this?